src/components/Notification/notification.scss

```diff
@@ -0,0 +1,76 @@
+.notification {
+  &-success {
+    color: var(--success-color);
+    border-left: 4px solid var(--toast-success-color);
+  }
+
+  &-device-change {
+    background: rgba(6, 14, 43, 0.9) !important;
+    width: 100% !important;
+    min-height: 36px;
+    padding: 8px !important;
+    .ant-notification-notice-message {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin: 0px;
+    }
+  }
+
+  &-error {
+    color: var(--danger-color);
+    border-left: 4px solid var(--danger-color);
+  }
+
+  &-warning {
+    color: var(--warning-color);
+    border-left: 4px solid var(--warning-color);
+  }
+
+  &-info {
+    color: var(--primary-color);
+    border-left: 4px solid var(--primary-color);
+  }
+
+}
+
+.ant-notification {
```

Comment:
can we add our custom css name later if some dependency comes with notification css may break